.github/workflows/update-codeclimate-coverage.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - name: Installing node.js
-        uses: actions/setup-node@v1 # Used to install node environment - XXX https://github.com/actions/setup-node
+        uses: actions/setup-node@v2.1.3 # Used to install node environment - XXX https://github.com/actions/setup-node
```

Comment:
@Demmonius Gotta check that :)

Reply:
setup-node v2 is still in beta, f*cking bot :joy: 
https://github.com/actions/setup-node/releases

Reply:
Yeah, so let's close this and we'll see once it's mature enough.